apps/launchpad/app/lib/utils/misc.tsx

```diff
@@ -93,3 +93,26 @@ export function truncateString(str: string, maxLength: number): string {
   }
   return `${str.slice(0, maxLength)}...`
 }
+
+export const truncateNumber = (balance: string | number): string => {
```

Comment:
More and more we have reason to create a shared utils package at this point, especially if we're moving toward having smart components in 1ui

@auroter @orbro vis